Extension/src/LanguageServer/client.ts

```diff
@@ -947,6 +949,9 @@ export class DefaultClient implements Client {
     }
 
     private createLanguageClient(allClients: ClientCollection): LanguageClient {
+        // do we need operating system?
+        const isFilePathHandlingCaseSensitivePersistent: PersistentWorkspaceState<string> = new PersistentWorkspaceState<string>("CPP.isFilePathHandlingCaseSensitivePersistent", "default");
+        let detectFilePathChange: boolean = false;
```

Comment:
A better name for this might be 'resetDatabase' since it reflects what will happen when it's true.